### PR TITLE
Added the new videos

### LINKED ISSUE #610: Add these new videos to the Semaphore website
https://youtu.be/WEKqycIMmLY

https://youtu.be/tx1Xglf07yE

### PR DESCRIPTION
**Pull request by dicethedev**
Fix the Add these new videos to the Semaphore website #610

### CODE REVIEW

File: apps/website/src/data/videos.json [hunk 1]
```diff
@@ -102,5 +102,21 @@
         "speakers": ["Vivian Plasencia"],
         "url": "https://youtu.be/4e4EAdu0WVg",
         "thumbnail": "https://img.youtube.com/vi/4e4EAdu0WVg/0.jpg"
+    },   
+    {
+        "title": "Semaphore in a Nutshell",
+        "eventName": "Devconnect 2023",
```
Comment (vplasencia) [inline apps/website/src/data/videos.json]:
```suggestion
        "eventName": "PROGCRYPTO - Devconnect 2023",
```

File: apps/website/src/data/videos.json [hunk 2]
```diff
@@ -102,5 +102,21 @@
...
+        "date": "2023-11-16",
+        "speakers": ["Cedoor"],
+        "url": "https://youtu.be/WEKqycIMmLY",
+        "thumbnail": "https://img.youtube.com/vi/4e4EAdu0WVg/0.jpg"
```
Comment (vplasencia) [inline apps/website/src/data/videos.json]:
```suggestion
        "thumbnail": "https://img.youtube.com/vi/WEKqycIMmLY/0.jpg"
```

File: apps/website/src/data/videos.json [hunk 3]
```diff
@@ -102,5 +102,21 @@
...
+    },
+    {
+        "title": "The Power of Anonymity",
```
Comment (vplasencia) [inline apps/website/src/data/videos.json]:
```suggestion
        "title": "Semaphore: The power of anonymity",
```

File: apps/website/src/data/videos.json [hunk 4]
```diff
@@ -102,5 +102,21 @@
...
+        "eventName": "ETHGlobal Circuit Breaker",
+        "date": "2024-02-02",
+        "speakers": ["Vivian Plasencia"],
+        "url": "https://youtu.be/tx1Xglf07yE",
+        "thumbnail": "https://img.youtube.com/vi/4e4EAdu0WVg/0.jpg"
```
Comment (vplasencia) [inline apps/website/src/data/videos.json]:
```suggestion
        "thumbnail": "https://img.youtube.com/vi/tx1Xglf07yE/0.jpg"
```

**Review by vplasencia — CHANGES_REQUESTED** [review]
Hey @dicethedev 

Thank you so much! Great work! ✨

I added some comments to update an event name, a title and the thumbnails.

You can click on `Commit Suggestion` to update them automatically.

Can you run `yarn prettier:write` and push the changes?

**Comment by dicethedev** [comment]
> yarn prettier:write

just push update @vplasencia

**Review by vplasencia — APPROVED** [review]
Thank you @dicethedev ! ✨

**Review by cedoor — APPROVED** [review]

**Pull request merged**